bedrock/base/templates/product-all-macros.html

```diff
@@ -84,10 +84,14 @@ <h4 class="not-found{% if builds %} hide{% endif %}">{{ _('No matching languages
 #}
 {% macro build_link(build, platform, tooltip) %}
   {% if build.platforms[platform] %}
-    <td class="download {{ platform }}"><a href="{{ build.platforms[platform].download_url }}" title="{{ tooltip }}"
-    {% if platform == 'android' %}data-link-type="download" data-download-os="Android"
-    {% elif platform == 'ios' %}data-link-type="download" data-download-os="iOS"
-    {% else %}data-link-type="download" data-download-os="Desktop"{% endif %}>{{ _('Download') }}</a></td>
+    <td class="download {{ platform }}">
+        <a href="{{ build.platforms[platform].download_url }}" title="{{ tooltip }}"
+           data-download-version="{{ platform }}" data-download-language="{{ build.name_en|lower}}"
+           {% if platform == 'android' %}data-link-type="download" data-download-os="Android"
```

Comment:
We can assume that `data-link-type="download"` will always be true here, so we can move it outside of the conditional, like you have for `data-download-version` and `data-download-language`.